### PR TITLE
🚸 Add link to about wnft page in index

### PR DESCRIPTION
**Pull request by williamchong**
image

image

image

### CODE REVIEW

File: src/components/NFTAboutPage/index.vue [hunk 1]
```diff
@@ -21,14 +21,29 @@
               leading-[1.25]
               text-center
             "
+          >
+            <span class="whitespace-nowrap">{{
+              $t('about_nft_page_title')
+            }}</span>
+            <br/>
```
Comment (nwingt) [inline src/components/NFTAboutPage/index.vue:28]: What is this br for?

File: src/pages/index.vue [hunk 2]
```diff
@@ -82,7 +82,13 @@
         'mt-[96px]',
       ]"
     >
-      <h2 class="text-[#3AB7A2] text-[48px] font-proxima font-[600]">{{ $t('home_section_articles_title') }}</h2>
+      <h2 class="text-[#3AB7A2] text-[48px] font-proxima font-[600] inline">{{ $t('home_section_articles_title') }}</h2>
+      <NuxtLink
+        class="mx-[6px] align-text-top text-medium-gray hover:text-like-cyan-dark text-[12px] cursor-pointer"
+        :to="localeLocation({ name: 'writing-nft-about' })"
+      >
+        ?
```
Comment (nwingt) [inline src/pages/index.vue:90]: Using a icon is better

File: src/pages/index.vue [hunk 3]
```diff
@@ -126,10 +132,16 @@
     </section>
 
     <section
-      v-if="!isInInAppBrowser"
-      class="mt-[88px] px-[24px]"
+      class="mt-[88px] px-[24px] flex gap-[16px]"
```
Comment (nwingt) [inline src/pages/index.vue:135]:
```suggestion
      class="mt-[88px] px-[24px] flex flex-wrap justify-center gap-[16px]"
```

File: src/pages/index.vue [hunk 4]
```diff
@@ -126,10 +132,16 @@
...
     >
+      <NuxtLink
+        class="flex items-center justify-center mt-[8px] text-medium-gray hover:text-like-cyan-dark text-[12px] leading-[5/3] underline transition-colors cursor-pointer"
```
Comment (nwingt) [inline src/pages/index.vue:138]: Should update the class for next link too
```suggestion
        class="text-medium-gray hover:text-like-cyan-dark text-[12px] leading-[5/3] underline transition-colors cursor-pointer"
```

**Comment by williamchong** [comment]
image
updated

**Review by nwingt — APPROVED** [review]

**Pull request merged**